pkgs/top-level/all-packages.nix

```diff
@@ -15039,6 +15039,8 @@ with pkgs;
   xfsprogs = callPackage ../tools/filesystems/xfsprogs { };
   libxfs = xfsprogs.dev;
 
+  xfs_undelete = callPackage ../tools/filesystems/xfs_undelete { };
```

Comment:
Consider moving the package to `pkgs/by-name/xf/xfs_undelete/package.nix`. I think it's only a matter of short time before the CI won't allow the non-by-name locations anymore.

Reply:
@h7x4, done, thank you!